videoinput.c

```diff
@@ -492,6 +558,9 @@ void cVideoInput::Close()
       cCamSlot *cs = m_Device->CamSlot();
       if (cs)
         cs->CancelActivation();
+      if (m_Receiver)
```

Comment:
I can't test it with more than one slot. That VDR code looks really nasty.

And I'm working on re-tune...